src/Framework/Framework/Binding/ActiveDotvvmProperty.cs

```diff
@@ -10,20 +10,20 @@
 
 namespace DotVVM.Framework.Binding
 {
-    public abstract class ActiveDotvvmProperty: DotvvmProperty
+    public abstract class ActiveDotvvmProperty : DotvvmProperty
     {
         public abstract void AddAttributesToRender(IHtmlWriter writer, IDotvvmRequestContext context, DotvvmControl control);
 
 
         public static ActiveDotvvmProperty RegisterCommandToAttribute<TDeclaringType>(string name, string attributeName)
         {
-            return DelegateActionProperty<ICommandBinding>.Register<TDeclaringType>(name, (writer, context, property, control) =>
-            {
-                var binding = control.GetCommandBinding(property) ?? throw new DotvvmControlException(control, $"Command binding expression was expected in {property}.") {
-                    RelatedProperty = property
-                };
-                var script = KnockoutHelper.GenerateClientPostBackScript(name, binding, control);
-                writer.AddAttribute(attributeName, script);
+            return DelegateActionProperty<ICommandBinding>.Register<TDeclaringType>(name, (writer, context, property, control) => {
+                var binding = control.GetCommandBinding(property);
```

Comment:
This should throw an exception if the property value is a constant. You can use `(ICommandBinding)control.GetValueRaw` or add a custom error check, but `GetCommandBinding` returns null if there is a constant.